lib/tasks/pact.rake

```diff
@@ -1,62 +1,30 @@
-unless Rails.env.production?
-  require 'pact/tasks'
+return if Rails.env.production?
 
-  desc "Verifies the pact files for latest release and master"
-  task "pact:verify" do
-    require 'pact/tasks/task_helper'
+require 'pact/tasks'
+require 'pact_broker/client/tasks'
+require 'pact/tasks/task_helper'
 
-    Pact::TaskHelper.handle_verification_failure do
-      Pact::TaskHelper.execute_pact_verify
-    end
-
-    unless ENV['USE_LOCAL_PACT'] # avoid running twice against the same pact file.
-      with_temporary_env('GDS_API_PACT_VERSION', "master") do
```

Comment:
Is it the case the we want to test against master _and_ the released version for `gds-api-adapters`? If not we should update the [docs](https://github.com/alphagov/publishing-api/blob/master/doc/pact_testing.md#how-the-tests-run-in-ci).

Reply:
Ah so that must be why it was set to run twice.

So going back through history. It was originally set to run the "latest" version: https://github.com/alphagov/publishing-api/commit/375233a81644151dd9f24bc41e554a0fa8ef97b8 which wasn't actually the released version but just the [most recent branch that was committed](https://github.com/alphagov/publishing-api/commit/f095211c3e3a3ac2dfbbcb059922880a1e387d66).

Hmm I don't think it makes sense to test against both master and released version - since if one fails you've got a failed build anyway. I'll see though whether I can change it to run against just released_version rather than master by default and update docs.

Thanks for catching the docs issue.